tox.ini

```diff
@@ -8,6 +8,7 @@ deps =
     -rrequirements-dev.txt
     -rrequirements.txt
 commands=
+    - pip install yelp-clog
```

Comment:
would probably be nice to have a comment here about why this is being installed - but since this is temporary i won't be too picky :p

Reply:
Added the comment.